liquibase-couchbase/src/test/java/liquibase/ext/couchbase/change/CreateBucketChangeTest.java

```diff
@@ -0,0 +1,96 @@
+package liquibase.ext.couchbase.change;
+
+import com.couchbase.client.core.msg.kv.DurabilityLevel;
+import com.couchbase.client.java.manager.bucket.BucketType;
+import com.couchbase.client.java.manager.bucket.CompressionMode;
+import com.couchbase.client.java.manager.bucket.ConflictResolutionType;
+import com.couchbase.client.java.manager.bucket.EvictionPolicyType;
+import common.TestChangeLogProvider;
+import liquibase.change.Change;
+import liquibase.changelog.ChangeSet;
+import liquibase.changelog.DatabaseChangeLog;
+import liquibase.ext.couchbase.change.utils.BucketCreationMapper;
+import liquibase.ext.couchbase.changelog.ChangeLogProvider;
+import liquibase.ext.couchbase.database.CouchbaseLiquibaseDatabase;
+import liquibase.ext.couchbase.statement.CreateBucketStatement;
+import liquibase.statement.SqlStatement;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static common.constants.ChangeLogSampleFilePaths.CREATE_BUCKET_CHANGE_TEST_XML;
+import static common.constants.TestConstants.TEST_BUCKET;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.internal.util.collections.Iterables.firstOf;
+
+class CreateBucketChangeTest {
+
+    private ChangeLogProvider changeLogProvider;
+
+    @BeforeEach
+    void setUp() {
+        CouchbaseLiquibaseDatabase db = mock(CouchbaseLiquibaseDatabase.class);
+        changeLogProvider = new TestChangeLogProvider(db);
+    }
```

Comment:
I think we can mock 1 time and create provider 1 time. Using @Mock and @InjectMocks annotations. We have such examples in code you can look at them.

And the same for such code below